### PR TITLE
Added a presistent theme toggler for basement dwellers

### PR DESCRIPTION
**Pull request by nate-moo**
Small update to visual system. uses localstorage to persistently store the data for the theme.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,7 +95,19 @@ <h1>Fujifilm X/GFX/HS Firmware Patcher</h1>
...
+	<div id="bottom">
+		<button onclick="toggleTheme()" id="theme">Theme Toggle</button>
+	</div>
 </body>
+<script>
+	"true" == localStorage.getItem("dark") ? 
+		document.body.classList = 'dark' : 
+		document.body.classList = '';
+
+	const toggleTheme = () => {
```
Comment (petabyt) [inline index.html]: Interesting way of declaring a function...

**Comment by petabyt** [comment]
Just tested, looks good. Not sure if it should go in `ui.js` or not, `index.html` is probably fine for now.

File: index.html [hunk 2]
```diff
@@ -60,9 +69,21 @@
 	.greyed {
 		color: #737272;
 	}
+
+	div#bottom {
+		/* position: absolute; */
```
Comment (petabyt) [inline index.html]: should be deleted

File: index.html (hunk 1 above)
Comment (nate-moo) [reply index.html]: yea... there are like 500 ways in javascript and that's one of them.

Comment (petabyt) [reply index.html]: Would prefer the classic way, with `function`

**Comment by nate-moo** [comment]
> Just tested, looks good. Not sure if it should go in `ui.js` or not, `index.html` is probably fine for now.

One of the reasons I have it inside of `index.html` is to make sure it loads first to prevent the screen from flickering before the script loads. if you want me to I can move it into `ui.js` and it should be fine.

File: index.html (hunk 1 above)
Comment (nate-moo) [reply index.html]: changed

**Pull request merged**